cEP-0000.md

```diff
@@ -1,17 +1,15 @@
-coala Enhancement Proposals
-===========================
+# coala Enhancement Proposals
 
-|Metadata|                                   |
-|--------|-----------------------------------|
-|cEP     |0000                               |
-|Version |2.2                                |
-|Title   |coala Enhancement Proposals        |
-|Authors |Lasse Schuirmann <lasse@gitmate.io>|
-|Status  |Active                             |
-|Type    |Process                            |
+| Metadata |                                            |
+| -------- | ------------------------------------------ |
+| cEP      | 0000                                       |
+| Version  | 2.2                                        |
+| Title    | coala Enhancement Proposals                |
+| Authors  | Lasse Schuirmann <mailto:lasse@gitmate.io> |
```

Comment:
Not sure if we want this, since GitHub already highlight email without `mailto:`

Reply:
Seems to be useful; see https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-no-auto-link-without-protocol